src/messages/responses/cache-sorted-set-get-scores.ts

```diff
@@ -0,0 +1,169 @@
+ * the following subtypes:
+ *
+ * - {Hit}
+ * - {Miss}
+ * - {Error}
+ *
+ * `instanceof` type guards can be used to operate on the appropriate subtype.
+ * @example
+ * For example:
+ * ```
+ * if (response instanceof CacheSortedSetGetScores.Error) {
+ *   // Handle error as appropriate.  The compiler will smart-cast `response` to type
+ *   // `CacheSortedSetGetScores.Error` in this block, so you will have access to the properties
+ *   // of the Error class; e.g. `response.errorCode()`.
+ * }
+ * ```
+ */
+export abstract class Response extends ResponseBase {}
+
+class _Hit extends Response {
+  public responses: CacheSortedSetGetScoreResponseType[] = [];
+
+  constructor(
+    scores: grpcCache._SortedSetGetScoreResponse._SortedSetGetScoreResponsePart[],
+    values: Uint8Array[]
+  ) {
+    super();
+    scores.forEach((score, index) => {
+      if (score.result === grpcCache.ECacheResult.Hit) {
+        this.responses.push(
+          new CacheSortedSetGetScoreResponse.Hit(score.score, values[index])
+        );
+      } else if (score.result === grpcCache.ECacheResult.Miss) {
+        this.responses.push(
+          new CacheSortedSetGetScoreResponse.Miss(values[index])
+        );
+      } else {
+        this.responses.push(
+          new CacheSortedSetGetScoreResponse.Error(
+            new UnknownError(score.result.toString()),
+            values[index]
+          )
+        );
+      }
+    });
+  }
+
+  /**
+   * Returns the data as a Map whose keys are byte arrays and values numbers.
+   * @returns {Map<Uint8Array, number>}
+   */
+  public valueMapUint8Array(): Map<Uint8Array, number> {
+    return this.responses.reduce((acc, response) => {
+      if (response instanceof CacheSortedSetGetScoreResponse.Hit) {
+        acc.set(response.valueUint8Array(), response.score());
+      }
+      return acc;
+    }, new Map<Uint8Array, number>());
+  }
+
+  /**
+   * Returns the data as a Map whose keys are utf-8 strings, decoded from the underlying byte arrays and values are numbers.
+   * @returns {Map<string, number>}
+   */
+  public valueMapString(): Map<string, number> {
+    return this.responses.reduce((acc, response) => {
+      if (response instanceof CacheSortedSetGetScoreResponse.Hit) {
+        acc.set(response.valueString(), response.score());
+      }
+      return acc;
+    }, new Map<string, number>());
+  }
+
+  /**
+   * Returns the data as a Map whose keys and values are utf-8 strings, decoded from the underlying byte arrays.
+   * This is a convenience alias for {valueMapStringString}.
+   * @returns {Map<string, number>}
+   */
+  public valueMap(): Map<string, number> {
+    return this.valueMapString();
+  }
+
+  /**
+   * Returns the data as a Record whose keys and values are utf-8 strings, decoded from the underlying byte arrays.
+   * This can be used in most places where an Object is desired.
+   * @returns {Record<string, number>}
+   */
+  public valueRecordStringNumber(): Record<string, number> {
```

Comment:
do we wanna remove Number here to be consistent?

Reply:
Yes absolutely. This slipped my mind

Reply:
Fixed in [358a0c6](https://github.com/momentohq/client-sdk-nodejs/pull/325/commits/358a0c6214074f9eeb4569fbc3e923102b673afb)